src/Commands/InstallCommand.php

```diff
@@ -116,6 +113,9 @@ public function handle(Filesystem $filesystem)
             $this->seed('VoyagerDummyDatabaseSeeder');
         }
 
+        $this->info('Setting up the hooks');
+        $this->call('hook:setup');
+
```

Comment:
Moved to execute the `hook:setup` command until after Voyager has run its own seeders.
Since if the `Admin` menu does not exists, the submenu `Hooks` will not be added.